pom.xml

```diff
@@ -84,7 +84,7 @@
     </modules>
     <profiles>
         <profile>
-            <id>default</id>
+            <id>merged-it</id>
```

Comment:
This profile does not seem to have any relation with the other `default` profiles declared in the individual integration test modules, as it uses a different activation. Seemed appropriate to rename it.

Reply:
agree

Reply:
agree